### PR TITLE
support FIPS better

### PR DESCRIPTION
**Pull request by lizan**
Signed-off-by: Lizan Zhou <lizan@tetrate.io>

### CODE REVIEW

File: linux-glibc/Dockerfile.rbe-fips [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Tetrate
```
Comment (mathetake) [inline linux-glibc/Dockerfile.rbe-fips:1]: nit: 2021? :slightly_smiling_face:

File: linux-glibc/build_container/getenvoy_linux_glibc_rbe_fips.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2019 Tetrate
```
Comment (mathetake) [inline linux-glibc/build_container/getenvoy_linux_glibc_rbe_fips.sh:3]: nit: 2021?

File: linux-glibc/build_container/getenvoy_linux_glibc_rbe_fips.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+set -e
+
+export DEBIAN_FRONTEND=noninteractive
+
+curl -sSL https://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz | \
+  tar Jx --strip-components=1 -C /usr/local
+
+# Force libc++ to be a static link by putting a linker script to do that.
+echo 'INPUT(-l:libc++.a -l:libc++abi.a -lm -lpthread)' > /usr/local/lib/libc++.so
```
Comment (mathetake) [inline linux-glibc/build_container/getenvoy_linux_glibc_rbe_fips.sh:25]: neat!

File: linux-glibc/build_container/getenvoy_linux_glibc_bazel.sh [hunk 4]
```diff
@@ -30,7 +30,10 @@ add-apt-repository \
    $(lsb_release -cs) \
    stable"
 
-apt-get update && apt-get -y install docker-ce docker-ce-cli containerd.io
+curl -fsSL https://apt.kitware.com/keys/kitware-archive-latest.asc | apt-key add -
+apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main"
+
+apt-get update && apt-get -y install docker-ce docker-ce-cli containerd.io cmake
```
Comment (musaprg) [inline linux-glibc/build_container/getenvoy_linux_glibc_bazel.sh:36]: nits: Is it possible to merge these lines to L23-25? https://github.com/tetratelabs/getenvoy-package/pull/96/files#diff-a1db49d1f80fa46ef76c7081ad0b378585ae5dcb9b7625a8bd46696889fa9040R23-R25 We might not need to care about the number of layers for this image, but just curious.

**Review by mathetake — APPROVED** [review]

File: linux-glibc/build_container/getenvoy_linux_glibc_bazel.sh (hunk 4 above)
Comment (lizan) [reply linux-glibc/build_container/getenvoy_linux_glibc_bazel.sh:36]: No, `software-properties-common` needed for `apt-add-repository`, and this install depends on those repositories.

**Pull request merged**